### PR TITLE
[INLONG-8302][Dashboard] Data synchronization source and sink type optimization

### PR DESCRIPTION
**Pull request by bluewang**
### Prepare a Pull Request
*(Change the title refer to the following example)*

- Title Example: [INLONG-XYZ][Component] Title of the pull request

*(The following *XYZ* should be replaced by the actual [GitHub Issue](https://github.com/apache/inlong/issues) number)*

- Fixes https://github.com/apache/inlong/issues/8302

### Motivation

*Explain here the context, and why you're making that change. What is the problem you're trying to solve?*

### Modifications

*Describe the modifications you've done.*

### Verifying this change
Other sink types also remove and add field form
image
image

image
image
image

### CODE REVIEW

**Comment by leezng** [comment]
Can the sink and source configurations be reused here?

**Review by leezng — COMMENTED** [review]
Can the sink and source configurations be reused here?

File: inlong-dashboard/src/plugins/syncSink/defaults/ClickHouse.ts [hunk 1]
```diff
@@ -0,0 +1,252 @@
...
+import { RenderList } from '@/plugins/RenderList';
+import i18n from '@/i18n';
+import { SinkInfo } from '../common/SinkInfo';
+import NodeSelect from '@/ui/components/NodeSelect';
+
+const { I18n } = DataWithBackend;
+const { FieldDecorator } = RenderRow;
+const { ColumnDecorator } = RenderList;
+
+export default class ClickHouseSink
```
Comment (healchow) [inline inlong-dashboard/src/plugins/syncSink/defaults/ClickHouse.ts:31]: There are too many duplicate files, please think about how to reuse them with existing Sink/Source files.

**Review by healchow — CHANGES_REQUESTED** [review]

File: inlong-dashboard/src/plugins/sources/defaults/index.ts [hunk 2]
```diff
@@ -24,16 +24,19 @@ export const allDefaultSources: MetaExportWithBackendList<SourceMetaType> = [
   {
     label: 'ALL',
     value: '',
+    isSync: false,
```
Comment (leezng) [inline inlong-dashboard/src/plugins/sources/defaults/index.ts:27]: isSync -> useInSync

**Review by leezng — APPROVED** [review]

**Review by healchow — APPROVED** [review]
LGTM

**Review by dockerzhang — APPROVED** [review]

**Pull request merged**